DatadogSDKBridge/Tests/DdRumTests.swift

```diff
@@ -169,22 +169,48 @@ internal class DdRumTests: XCTestCase {
 
         XCTAssertEqual(mockNativeRUM.calledMethods.count, 2)
 
-        XCTAssertEqual(mockNativeRUM.calledMethods.first, .addResourceMetrics(resourceKey: "resource key",
-                                                                              fetch: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 0), end: nanoTimeToDate(timestampNs: 13)),
-                                                                              redirection: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 1), end: nanoTimeToDate(timestampNs: 2)),
-                                                                              dns: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 3), end: nanoTimeToDate(timestampNs: 4)),
-                                                                              connect: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 5), end: nanoTimeToDate(timestampNs: 6)),
-                                                                              ssl: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 7), end: nanoTimeToDate(timestampNs: 8)),
-                                                                              firstByte: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 9), end: nanoTimeToDate(timestampNs: 10)),
-                                                                              download: MockNativeRUM.Interval(start: nanoTimeToDate(timestampNs: 11), end: nanoTimeToDate(timestampNs: 12)),
-                                                                              responseSize: nil))
+        XCTAssertEqual(
```

Comment:
btw, I'm curious - is there a way to auto-format the code when line limit is reached? neither Xcode, nor linter notify when it happens.

Reply:
Xcode doesn't have in-build linter and we use external tool - `swiftlint`. For what I see it is not properly configured in the `bridge-ios` as it is not called when compiling unit tests (I will fix it). Although `swiftlint` has `--auto-format` option, we don't use it - it doesn't integrate well with Xcode and often interrupts compilation by modifying files and resulting with compilation error due to "file was modified while building" issue.

Reply:
Thanks, good to know. It probably means I can run `--auto-format` once before I do commit? Do we accept default code style defined with `swiftlint`?

Reply:
Sorry, I don't know which rules it will use for `--auto-format`. Normally, what works best is just following linter warnings emitted in Xcode while writing code as on the picture below. Unfortunately, this doesn't work out-of-the-box in `bridge-ios` and it requires [manual configuration](https://github.com/DataDog/dd-bridge-ios/blob/main/CONTRIBUTING.md#running-the-tests). I created `RUMM-1460` to fix this project structure so its development is seamless and doesn't require much Xcode knowledge.

<img width="1275" alt="Screenshot 2021-06-29 at 10 22 48" src="https://user-images.githubusercontent.com/2358722/123764237-e3aeb780-d8c4-11eb-8ed1-28a975973e75.png">